packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapController.java

```diff
@@ -107,7 +109,7 @@ class GoogleMapController
   private List<Map<String, ?>> initialTileOverlays;
   // Null except between initialization and onMapReady.
   private @Nullable String initialMapStyle;
-  private @Nullable String lastStyleError;
+  private boolean lastSetStyleSucceeded;
```

Comment:
This seems like a semantic change I would not have expected.

Reply:
This is for: https://github.com/flutter/packages/pull/6967/files#diff-b884b5356d3b2ce3b472fc6c979dfee52603cec7e87a4b8c31bd04f42893d80eR898-R902

It's not necessary as part of this PR, just an opportunistic move of logic that didn't need to be in Java over to the Dart side. But if I didn't do this now, then I'd need to add complexity to the Pigeon API definition just to carry a hard-coded string over from the Java side.